File: app/src/main/java/com/shua/likegank/data/entity/Girl.java
package com.shua.likegank.data.entity;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Girl extends RealmObject {
    @PrimaryKey
    public String _id;
    public String dec;
    public String url;

    public Girl() {
    }

    public Girl(String _id, String dec, String url) {
        this._id = _id;
        this.dec = dec;
        this.url = url;
    }
}

File: app/src/main/java/com/shua/likegank/data/entity/Girl.kt
package com.shua.likegank.data.entity

import io.realm.kotlin.types.RealmObject
import io.realm.kotlin.types.annotations.PrimaryKey

open class Girl : RealmObject {
    @PrimaryKey
    var _id: String = ""
    var dec: String = ""
    var url: String = ""

    constructor()

    constructor(_id: String, dec: String, url: String) {
        this._id = _id
        this.dec = dec
        this.url = url
    }
}